cardano-crypto-class/src/Cardano/Crypto/Hash/Class.hs

```diff
@@ -183,7 +184,7 @@ hashFromBytesAsHex hexrep
 
 
 instance Show (Hash h a) where
-  show = show . hashToBytesAsHex
```

Comment:
Here `show` is called as `show @ByteString` which returns a string literal in a string.